core/server/master/src/main/java/alluxio/master/DefaultAlluxioMaster.java

```diff
@@ -313,6 +313,11 @@ private void registerServices(TMultiplexedProcessor processor, Map<String, TProc
     }
   }
 
+  /**
+   * Starts the Thrift RPC server. The AlluxioMaster registers the Services of registered
+   * {@link Master}s and meta services to  multiplexed processor, then creates master thrift
```

Comment:
to multiplexed -> to a multiplexed
creates master service -> creates the master service